setup.py

```diff
@@ -14,7 +14,7 @@
     author_email="<mkarimiziarani@ua.edu>",
     description=DESCRIPTION,
     packages=find_packages(),
-    install_requires=['dateutil', 'itertools', 'time', 'os'],
+    #install_requires=['dateutil', 'itertools', 'time', 'os'],
```

Comment:
Could we just delete the line? Do we have other commented code still in the script that relied on these libraries?